backend/app/mailers/offer_mailer.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class OfferMailer < ApplicationMailer
+    # default from comes from config environment
+    layout 'mailer'
+
+    def contract_email(assignment, link)
+        @offer = assignment.active_offer
+        @url = link
+        mail(to: assignment.applicant.email, subject: 'TA Position Offer')
```

Comment:
We could add the position code in the subject. That way when they hit reply, we can tell the emails apart...